regions/io/crtf/new_casa_tests/casa_mask_test.py

```diff
@@ -0,0 +1,86 @@
+import pickle
+import tempfile
+import pytest
+
+from astropy import coordinates, units as u
+
+from ..write import write_crtf
+
+from ....shapes.ellipse import EllipseSkyRegion
+
+try:
+    from casatools import image, simulator
+    from casatasks import tclean
```

Comment:
Just a note for us as we build this - importing `casatasks` will import matplolib and try to start a background, so we need to enable a headless X-server on travis.